components/editor/toolbar.md

```diff
@@ -137,10 +139,12 @@ This example shows how you can keep adding tools to the toolbar to get the desir
 
 ````
 @using Telerik.Blazor.Components.Editor
+@* Avoid ambiguous reference with SVG icons *@
+@using EditorNS = Telerik.Blazor.Components.Editor;
```

Comment:
This sample is not working. Add the namespace to all used tools otherwise, they are not recognized (e.g. EditorButtonGroup -> EditorNS.EditorButtonGroup).